src/base/trafficGraph.js

```diff
@@ -371,6 +374,10 @@ class TrafficGraph extends EventEmitter {
   setState (state, force, parentState) {
     let updatedState = false;
     if (state && Object.keys(state).length > 0) {
+      // it's important to update entryNode when we're switching views
+      if (state.entryNode) {
+        this.entryNode = state.entryNode;
+      }
```

Comment:
`setState` method in globalTrafficGraph were not passing `entryNode` to super.setState that's why entryNode was sometimes undefined even if present in trafficData.